identity-service/src/notifications/sendNotificationEmails.js

```diff
@@ -22,6 +22,31 @@ const loggingContext = {
   job: 'processEmailNotifications'
 }
 
+const getUserIdsWithUnseenNotifications = async ({ userIds, gtTimeStamp }) => {
+  const [notificationUserIds, solanaNotificationUserIds] = await Promise.all([
+    models.Notification.findAll({
+      attributes: ['userId'],
+      where: {
+        isViewed: false,
+        userId: { [ models.Sequelize.Op.in ]: userIds },
+        timestamp: { [models.Sequelize.Op.gt]: gtTimeStamp }
+      },
+      group: ['userId']
+    }),
+    models.SolanaNotification.findAll({
+      attributes: ['userId'],
+      where: {
+        isViewed: false,
+        userId: { [ models.Sequelize.Op.in ]: userIds },
+        createdAt: { [models.Sequelize.Op.gt]: gtTimeStamp }
+      },
+      group: ['userId']
+    })
+  ])
+
+  return notificationUserIds.concat(solanaNotificationUserIds).map(x => x.userId)
```

Comment:
Do you need to cull duplicates here?

Reply:
Should be fine, immediately added to a set afterwards